src/Controls/src/Core/IndicatorView/IndicatorView.cs

```diff
@@ -152,7 +152,7 @@ static void UpdateIndicatorLayout(IndicatorView indicatorView, object newValue)
 			{
 				indicatorView.IndicatorLayout = new IndicatorStackLayout(indicatorView) { Spacing = DefaultPadding };
 			}
-			else if (indicatorView.IndicatorLayout == null)
+			else if (indicatorView.IndicatorLayout != null)
```

Comment:
```suggestion
			else if (indicatorView.IndicatorLayout is not null)
```

Can you add a unit test as well?

Reply:
sure